doc/source/advanced.rst

```diff
@@ -844,3 +844,136 @@ Of course if you need integer based selection, then use ``iloc``
 .. ipython:: python
 
    dfir.iloc[0:5]
+
+Miscellaneous indexing FAQ
+--------------------------
+
+Integer indexing
+~~~~~~~~~~~~~~~~
+
+Label-based indexing with integer axis labels is a thorny topic. It has been
+discussed heavily on mailing lists and among various members of the scientific
+Python community. In pandas, our general viewpoint is that labels matter more
+than integer locations. Therefore, with an integer axis index *only*
```

Comment:
Yes, you are right. Originally this was a part about `ix`, and then this made more sense (the ix was replaced by loc after the deprecation)

I didn't really review the content of these parts, only moved them (apart from the comment that were already made on the other PR), but good to do that now as well.